plugins/rtmp-services/data/services.json

```diff
@@ -1145,6 +1145,28 @@
                 "max audio bitrate": 192
             }
         },
+        {
+            "name": "EventLive.pro",
+            "alt_names": ["EventLive", "Event Live"],
+            "servers": [
+                {
+                    "name": "Default",
+                    "url": "rtmp://go.eventlive.pro/live"
+                }
+            ],
+            "recommended": {
+                "keyint": 2,
+                "profile": "baseline",
+                "max video bitrate": 3000,
+                "max audio bitrate": 192,
+                "supported resolutions": [
+                    "1920x1080",
+                    "1280x720"
+                ],
+                "max fps": 30,
+                "x264opts": "tune=zerolatency scenecut=0"
```

Comment:
tune=zerolatency also produces much worse quality, is saving ~1 second of encoder latency really necessary?